app/src/main/java/ca/rmen/android/poetassistant/Tts.java

```diff
@@ -111,13 +127,15 @@ public void stop() {
     public void shutdown() {
         if (mTextToSpeech != null) {
             mTextToSpeech.shutdown();
+            mTtsStatus = TextToSpeech.ERROR;
```

Comment:
`shutdown()` still isn't used anywhere, but if somebody it's used, it will be better to reset `mTtsStatus` :)